docs/devguide/metricset-details.asciidoc

```diff
@@ -102,7 +101,10 @@ The first `fields.yml` file is located at the root of the repository and it is u
 To make sure the Elasticsearch template is correct, it's important to keep this file up-to-date
 with all the changes. 
 
-The second `fields.yml` file is under `module/{module}/_meta/fields.yml` and it contains
+[float]
+===== metricbeat/{module}/_meta/fields.yml
+
+The second `fields.yml` file is under `metricbeat/module/{module}/_meta/fields.yml` and it contains
 the general top level structure for all metricsets. Normally you only need to modify the description in this file.
 
 Here an example for the `fields.yml` file from the MySQL module.
```

Comment:
Sorry, I missed this before, but there is a missing word here. Should say "Here is an example" or "Here's an example"

Reply:
Sorry about that :)